src/_data/global.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    random() {
+      const segment = () => {
+        return (((1 + Math.random()) * 0x10000) | 0).toString(16).substring(1);
+      };
+      return `${segment()}-${segment()}-${segment()}`;
+    },
+    now: Date.now()
+  };
```

Comment:
What are these used for?

Reply:
Removed. Not being currently used function.